packages/web/src/pages/chat-page/components/ChatMessageTrack.tsx

```diff
@@ -86,7 +86,7 @@ export const ChatMessageTrack = ({
   })
 
   useEffect(() => {
-    if (track && uid) {
+    if (track && uid && !track.is_delete && !track.is_unlisted) {
```

Comment:
i think this is correct for deleted tracks.
but hidden tracks is a different story - i think for chats we def want to be able to unfurl them. this is a good way for artists to share hidden tracks with other users.
not sure we want the same behavior for comments tho.. maybe worth checking w design

Reply:
pretty sure `is_unlisted` is what we use for hidden tracks. not sure how the unfurl was working with this code tho??

Reply:
but if it works it works 🤷